proto/api_v3/query_service.proto

```diff
@@ -48,13 +48,13 @@ message TraceQueryParameters {
   // Attributes are matched against Span and Resource attributes.
   // At least one span in a trace must match all specified attributes.
   map<string, string> attributes = 3;
-  // Span min start time.
+  // Span min start time in. RFC-3339ns format. Required.
```

Comment:
> RFC-3339ns

A link would be useful, but not sure what "format" means in this case in proto, since it's determined by google.protobuf.Timestamp definition.

Reply:
If you're referring to the format in JSON, then we should phrase it differently. Doesn't grpc-gateway allow generating swagger definition? Ideally we should channel these types of comments there.

Reply:
> Doesn't grpc-gateway allow generating swagger definition? Ideally we should channel these types of comments there.

I didn't find annotations for this.